common/djangoapps/pipeline_mako/tests/test_render.py

```diff
@@ -49,6 +50,16 @@ def setUpClass(cls):
         Create static assets once for all pipeline render tests.
         """
         super(PipelineRenderTest, cls).setUpClass()
+        # Ensure that the npm requirements are always installed before updating static assets.
+        # Restore the original environment var value after installing npm requirements.
+        prereq_install_val_orig = os.environ.get('NO_PREREQ_INSTALL')
+        os.environ['NO_PREREQ_INSTALL'] = 'False'
+        call_task('pavelib.prereqs.install_node_prereqs')
+        if prereq_install_val_orig is None:
+            del os.environ['NO_PREREQ_INSTALL']
+        else:
+            os.environ['NO_PREREQ_INSTALL'] = prereq_install_val_orig
```

Comment:
Should the environment reset be handled via `addCleanup()` so it gets run even if the test fails or hits an error?

Reply:
I'll try putting the env reset in a `tearDownClass` since it's overridden in `setUpClass`.